modules/videoio/test/test_camera.cpp

```diff
@@ -72,6 +72,27 @@ TEST(DISABLED_videoio_camera, basic)
     capture.release();
 }
 
+// Test that CAP_PROP_CONVERT_RGB remain to false (default is true) after other supported property are set.
+// The test use odd value to be almost sure to trigger code responsible for recreating the device.
+TEST(DISABLED_videoio_camera, dshow_convert_rgb_persistency)
+{
+    VideoCapture capture(CAP_DSHOW);
+    ASSERT_TRUE(capture.isOpened());
+    ASSERT_TRUE(capture.set(CAP_PROP_CONVERT_RGB, false));
```

Comment:
Please avoid [build warning](http://pullrequest.opencv.org/buildbot/builders/precommit_windows64/builds/28017) here:

```
ASSERT_TRUE(capture.set(CAP_PROP_CONVERT_RGB, 0/*false*/));
```

Reply:
bf6460cf200fba60d11b436001e88f8443e4b056 Should fix it

Reply:
913739bbe6c2c235fafa8a7c66940c91095c2e63 should better work